ginkgo-kubernetes-all.Jenkinsfile

```diff
@@ -84,6 +85,9 @@ pipeline {
                     "K8s-1.9":{
                         sh 'cd ${TESTDIR}; K8S_VERSION=1.9 ginkgo --focus=" K8s*" -v --failFast=${FAILFAST}'
                     },
+                    "K8s-1.10":{
```

Comment:
I know that it is still wip, but running 3 different version of Kubernetes in the same stage is not a good idea IMHO.  I would use two k8s versions per stage.

Reply:
Should I create a new stage? :thinking:

Reply:
I would add that one in the non-release ones, and change the names, mora than two are too much for the host IMHO